ceno_zkvm/src/chip_handler/general.rs

```diff
@@ -321,8 +321,8 @@ impl<'a, E: ExtensionField> CircuitBuilder<'a, E> {
     pub fn namespace<NR: Into<String>, N: FnOnce() -> NR, T>(
         &mut self,
         name_fn: N,
-        cb: impl FnOnce(&mut CircuitBuilder<E>) -> Result<T, ZKVMError>,
-    ) -> Result<T, ZKVMError> {
+        cb: impl FnOnce(&mut CircuitBuilder<E>) -> T,
```

Comment:
@lispc Here's a simple realisation: the `namespace` function as written actually works for any type `T`, not just for `Result`.

Realising that, lets us remove plenty of unnecessary `OK(..)` and `.unwrap()` later down.

It's useful to do this, because `Result` types should be used when we actually have something that can go wrong and should be handled; instead of being dealt with via `.unwrap()` thoughout.  (If all the 'error handling' you ever gonna do is via `.unwrap()`, you might as well `panic!()` instead of creating an `Err` in the first place, at least that gives more useful stack traces.)